Export log to JSON (#4527)
DataDog/documentation: content/en/logs/explorer/_index.md

The **View in context** button updates the search request in order to show you the log lines dated just before and after a selected log—even if they don't match your filter. This context is different according to the situation, as Datadog uses the `Hostname`, `Service`, `filename`, and `container_id` attributes, along with tags, in order find the appropriate context for your logs.

> Copy the log content as a JSON into clipboard through the **Export** button.

Reviewer: ```suggestion
Copy the log content JSON to the clipboard through the **Export** button.
```
